### PR TITLE
Add hazelcast official helm repository

### PR DESCRIPTION
**Pull request by mesutcelik**
This PR adds hazelcast official chart repository to the documentation.

### CODE REVIEW

**Comment by hz-highfive** [comment]
The job `jet-oss-pr-builder` of your PR failed ([log](
https://j-artifacts.s3.amazonaws.com/jet-oss-pr-builder/2982/build_log.txt), [artifacts](https://j-artifacts.s3.amazonaws.com/jet-oss-pr-builder/2982/jet-oss-pr-builder-2982.zip)). 
Through arcane magic we have determined that the following fragments from the build log may contain information about the problem.
<details>
<summary>Click to expand the log file</summary>
<pre>
--------------------------
-------TEST FAILURE-------
--------------------------
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   ObservableResultsTest.configureCapacity:521->assertThrowsException:606 expected:<class java.lang.IllegalStateException> but was:<class java.lang.RuntimeException>
[INFO] 
[ERROR] Tests run: 2217, Failures: 1, Errors: 0, Skipped: 12
[INFO] 

[ERROR] There are test failures.
</details>
</pre>

**Comment by cangencer** [comment]
I think this page also needs editing: https://jet-start.sh/docs/next/get-started/intro

**Comment by cangencer** [comment]
@mesutcelik thanks for PR, @frant-hartm could you review ?

File: site/docs/operations/kubernetes.md [hunk 1]
```diff
@@ -43,27 +43,36 @@ enterprise versions also for Hazelcast Jet Management Center.
 
 ### Installing the Chart
 
+Hazelcast Official Chart Repository is hosted at https://github.com/hazelcast/charts
```
Comment (frant-hartm) [inline site/docs/operations/kubernetes.md:46]: This confused me that the charts repo moved away from AWS, but it is actually where the **sources** are hosted. So maybe word this as 

>Hazelcast Official Chart Repository has sources hosted at https://github.com/hazelcast/charts

or similar.

**Comment by mesutcelik** [comment]
Thank @frant-hartm , updated the doc.
@cangencer I am fine waiting for 4.2.

**Pull request merged**